pkgs/servers/misc/irrd/default.nix

```diff
@@ -19,31 +20,23 @@ let
         };
         doCheck = false;
       });
-      starlette = prev.starlette.overridePythonAttrs (oldAttrs: rec {
-        version = "0.20.4";
+      flask-sqlalchemy = prev.flask-sqlalchemy.overridePythonAttrs (oldAttrs: rec {
+        version = "2.5.1";
```

Comment:
Would you mind adding a hint, why we do an override here? 
Probably "irrd needs an v2 version of flask-sqlachemy, we package a v3"?

Reply:
Whoops that is not actually required